### ISSUE TITLE
IN and NOT IN filter

### ISSUE THREAD

**Issue by FCMHUB** [igorbenav/fastcrud#60]
Thanks for this awesome package.

**Is your feature request related to a problem? Please describe.**
It should be possible to filter records using the **IN** and **NOT IN** operators in the **get** and **get_multi** functions.

**Describe the solution you'd like**
```python
db_asset = await crud_users.get(
        db=db, 
        schema_to_select=User, 
        return_as_model=True, 
        id=id, 
        filter=User.id.not_in(ids),
        is_deleted=False,
)
```

It would be great to utilize logical operators such as `and_` and `or_`.

**Describe alternatives you've considered**
Currently, one must rely on SQLAlchemy methods to execute such filtering operations.

```python
smt = select(User).where(User.reference_id == ref_id).filter(User.id.not_in(ids))
```

**Comment by JakNowy** [comment]
Please see [my latest PR](https://github.com/igorbenav/fastcrud/pull/57)

`and_` is handled by default by chaining filters. I also tried `or_` but it was slightly more complex. `not_in_` could be an additional option in my PR actually.

**Comment by igorbenav** [comment]
closed by #57

**Comment by igorbenav** [comment]
closed by #57
